README.md

```diff
@@ -10,30 +10,53 @@
 > **NOTE** The rtc-tools repository has been migrated from gitlab to here;
 see [migration from gitlab](#migration-from-gitlab).
 
-RTC-Tools is an open-source Python package for simulation and optimization of cyber-physical systems.  It can be used for model-predictive control and operational decision making of water and energy systems.  It is developed and maintained by Deltares, in collaboration with partners.
+## Project Overview
 
-RTC-Tools offers the following functionalities:
+RTC-Tools is an open-source Python package designed to model, simulate, and optimize time-dependent systems with interconnected components. It is part of [LF Energy](https://lfenergy.org/).
```

Comment:
Let's use the same sentence here are as in `index.rst`:
"RTC-Tools is an open-source Python package designed to model, simulate, and optimize networks or portfolios of assets, such as  reservoirs, pumps, renewables, and batteries. It is part of `LF Energy <https://lfenergy.org/>`_."